src/proxy-ui-api/src/test/java/org/niis/xroad/restapi/converter/LocalGroupConverterTest.java

```diff
@@ -60,7 +61,9 @@ public String getMemberName(ClientId identifier) {
                 return MEMBER_NAME_PREFIX + identifier.getMemberCode();
             }
         };
-        clientConverter = new ClientConverter(globalConfFacade);
+        ClientId ownerId = ClientId.create("XRD2", "GOV", "M4");
```

Comment:
Not related to this task directly but it seems that this test class does not require Spring context to run -> could remove the annotations `@RunWith(SpringRunner.class)` and `@SpringBootTest` [above](https://github.com/nordic-institute/X-Road/pull/364/files#diff-61310cd629adaaaad39a3c75c48e2d6aR47-R48) to make the test run  much faster. But IIRC there already is a task on the works to improve our tests?

Reply:
Actually I don't think we have a ticket? I have just worked on the issue outside of ticketing. Maybe would be good to create one.

Reply:
I can create the task